design/button.behavior.md

```diff
@@ -0,0 +1,94 @@
+# Button Behavior
+
+Button behavior is the high level representation of how an MRE actor should behave as a button.  This behavior consists of two actions
+that occur on the object when it is targeted and when it is activated or clicked.  As part of these interactions, the point of targeting
+of the button will be passed along as additional data to the app for consumption.
+
+Button Behavior Actions:
+- `hover` - The action that occurs when the button is targeted by the host app input system. 
+    - Hover states: `enter`, `hovering`, `exit`
+- `click` - The action that occurs when a complete click has happened on the button consisting of both a button pressed and button released event. 
+    - Click states: None.  This is a single atomic event.
+- `button` - The action that occurs when the button interaction state has changed from being pressed and then released.
+    - Button states: `pressed`, `holding`, `released`
+    
+Button Event Data:
+- `targetedPoints` - The collection of target points for the current state event of the action.  This collection is of `PointData` that contains both app space and local target object space versions of the points.
```

Comment:
Does the collection have temporal points? It is unclear why there would be multiple points for an event

Reply:
Oh, this is explained later

Reply:
I would add: one or more points depending on event type